index.html

```diff
@@ -2,8 +2,8 @@
 <html lang="en">
 <head>
 	<meta charset="UTF-8">
-    <meta http-equiv="Content-Security-Policy" content="default-src https:">
```

Comment:
This meta tag with the content security policy was stopping the css and js from loading chrome on my local machine so I had to remove it. 

Is this something that you would like to keep?

Reply:
We can remove this, no problem